lib/src/internet_check_option.dart

```diff
@@ -1,5 +1,7 @@
 part of internet_connection_checker_plus;
 
+typedef ResponseStatusFn = bool Function(http.Response response);
+
```

Comment:
Please add documentation for this public definition.

Reply:
sure will do

Reply:
You mistakenly used `//` instead of `///` here.